models/base/scratch/default/contexts/snowplow_mobile_base_app_context.sql

```diff
@@ -1,6 +1,5 @@
 {{ 
-  config(materialized='view',
```

Comment:
Sorry forgot to mention on the last pass given these are now tables we should add a sort and dist key. I would suggest `dist=root_id`, `sort=root_tstamp`.